packages/dashboard/src/pages/templates/index.page.tsx

```diff
@@ -76,9 +71,35 @@ export const getServerSideProps: GetServerSideProps<PropsWithInitialState> =
     const templates = await findMessageTemplates({
       workspaceId,
     });
+    const journeys = await prisma().journey.findMany({
+      where: { workspaceId },
```

Comment:
We need to filter by `DBResourceType` equals `Declarative` here. Also can you parallelize this call with `Promise.all` with the `findMessageTemplates` call?